### DISCUSSION TITLE
How to update add-on

### DISCUSSION THREAD

**Discussion by ronaldt80** [fuatakgun/eufy_security#57] [Q&A] [answered]
Hi, I use the add-on in HA, any thoughts on how to update to the latest image of bropat? I have the feeling im stuck at v. 3.3

**Comment by fuatakgun** [comment, marked as answer]
I am still testing new version before releasing it, i had found an issue about p2p stream on add on, we first need to figure out it before releasing. You will get a notification from home assistant when it is released.

Anything you are waiting specifically?

**Reply by ronaldt80** [reply]
The p2p stream. Was hoping bropats new version provides better /more stable performance. 

Wired Doorbell works sometimes but performance is unstable. Negotiation with the add-on/ integration on codec takes a lot of time and results are unpredictable (works less than 10% of the time).

My eufycam 2c (on rtsp) works like a charm by the way! Will try to integrate it into frigate next week and run some ai on it.

**Reply by fuatakgun** [reply]
You are using rtsp add on right? it is better in terms of stability and latency.

Moreover, what are possible issues you are having? We can have one debug session?

**Reply by fuatakgun** [reply]
I will track open issues here: https://github.com/fuatakgun/eufy_security/issues/59

On the other hand, I am still having 5 seconds delay for P2P streaming, did not notice any improvement
